lib/ravelin/version.rb

```diff
@@ -1,3 +1,3 @@
 module Ravelin
-  VERSION = "0.1.5"
+  VERSION = "0.1.6"
```

Comment:
We're just about to do some work on this Gem. Would you mind squashing the version bump out so we can decouple this PR from the version we release it under?